packages/eas-cli/src/credentials/manager/ManageIosBeta.ts

```diff
@@ -274,6 +280,26 @@ export class ManageIosBeta implements Action {
         }
         return;
       }
+      case ActionType.RemoveProvisioningProfile: {
```

Comment:
We should keep this file as small as possible, so maybe consider either
- extracting cases like this as separate classes
- converting runAccountSpecificActionAsync into separate class

not necessarily in this PR, but I think this file is reaching the size that we should consider that

Reply:
Makes sense :)